utils/doclint/generateDotnetApi.js

```diff
@@ -461,9 +461,17 @@ function renderMethod(member, parent, output, name) {
 
   let parseArg = (/** @type {Documentation.Member} */ arg) => {
     if (arg.name === "options") {
-      arg.type.properties.forEach(prop => {
-        parseArg(prop);
-      });
+      arg.type.properties
+        .sort((a, b) => {
```

Comment:
I don't think your comparator is transitive. Try it on ['a', 'force', 'b', 'timeout', 'noWaitAfter']. I always use weights when I encounter a situation like this.

Reply:
How about now?

```
const fixedArguments = ['timeout', 'force', 'noWaitAfter'];

const sort = (a, b) => {
    return fixedArguments.indexOf(a) - fixedArguments.indexOf(b);
};

console.log(['a', 'force', 'b', 'timeout', 'noWaitAfter'].sort(sort));
console.log(['b', 'force', 'a', 'timeout', 'noWaitAfter'].sort(sort));
console.log(['b', 'force', 'a', 'timeout', 'c', 'noWaitAfter'].sort(sort));
console.log(['a', 'force', 'c', 'noWaitAfter', 'b', 'timeout'].sort(sort));
```